File: lab2/lab2q3.cpp
#include <iostream>
#include <string>
    
#include <sstream>
 

auto main() -> int
{

    std::string lineInput;
    std::string output_string;

	while ( getline( std::cin, lineInput ) )
	{
        int num {0};
        std::istringstream ( lineInput ) >> num;
        
        if (num % 3 == 0 and num % 5 == 0){
            auto output_string = std::to_string( num ) + std::string("\t") + std::string("Fizz  Buzz");
            std::cout << output_string << std::endl;

        } else if  (num % 3 == 0) {
            auto output_string = std::to_string( num ) + std::string("\t") + std::string("Fizz");
            std::cout << output_string << std::endl;

        } else if (num % 5 == 0){
            auto output_string = std::to_string( num ) + std::string("\t") + std::string("Buzz");
            std::cout << output_string << std::endl;

        } else if (num % 5 != 0 and num%3 !=0) {
            auto output_string = std::to_string( num );
            std::cout << output_string << std::endl;
        }

    }
}

File: lab2/lab2q3.py
import sys

for current_line in sys.stdin:
    
    stripped_line = current_line.strip()

    num = int ( stripped_line)

    if num % 3 == 0 and num % 5 == 0:
         output_string = stripped_line + "\t" + "Fizz  Buzz" 
    elif num % 3 == 0:
        output_string = stripped_line + "\t" + "Fizz"
    elif num % 5 == 0:
        output_string = stripped_line + "\t" + "Buzz"
    elif num % 5 != 0 and num%3 !=0:
        output_string = stripped_line 
    
    print( output_string)




